### DISCUSSION TITLE
inpaint pipelines

### DISCUSSION THREAD

**Discussion by heorhiikalaichev** [finegrain-ai/refiners#187] [Q&A] [answered]
Hi!

Thank you for your great job!!

Do you provide a possibility to use inpainting pipelines (StableDiffusionInpaintPipeline)?

**Comment by deltheil** [comment, marked as answer]
Hi @heorhiikalaichev - yes, we do support inpainting. You can find an example with SD 1.5 part of the test suite: https://github.com/finegrain-ai/refiners/blob/7839c54ae8913c6f19b701b481535ca9a0dec540/tests/e2e/test_diffusion.py#L764-L784

(Also, the weights conversion script handles `runwayml/stable-diffusion-inpainting` by default: https://github.com/finegrain-ai/refiners/blob/7839c54ae8913c6f19b701b481535ca9a0dec540/scripts/prepare_test_weights.py#L631)

Let us know if you have any other questions!
